packages/block-library/src/gallery/tiles.native.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { Children } from '@wordpress/element';
+
+const containerStyle = {
+	flexDirection: 'row',
+	flexWrap: 'wrap',
+};
+
+const tileStyle = {
+	overflow: 'hidden',
+	flexDirection: 'row',
+	alignItems: 'center',
+	borderColor: 'transparent',
+};
```

Comment:
These work, but if we're not going to use scss files, we should at least use `StyleSheet.create` to create these objects, as it adds some extra validation and type hinting.

Reply:
Using `rowLength - 1` as a denominator can be problematic for rows that only have one tile (divide by zero). The underlying problem I've struggled with in simplifying this further is that the left and right spacings for a given tile are not linearly dependent: i.e. right border cannot be expressed solely as a function of the spacing and the left border. Instead, the right border is complementary with the left border of the _next_ tile (hence the fencepost calculations). I've added an explanation in the code comment.